generators/client/templates/angular/src/main/webapp/app/_polyfills.ts

```diff
@@ -1,3 +1,27 @@
 /* tslint:disable */
+// Polyfills
 import 'reflect-metadata/Reflect';
+// import 'core-js/es6';
+// Added parts of es6 which are necessary for your project or your browser support requirements.
+import 'core-js/es6/symbol';
```

Comment:
Also we are generating es5 code with webpack so most of this shouldn't be needed at all in the first place. I think this needs further investigation